analyzer/analyzer.go

```diff
@@ -54,6 +54,48 @@ func newPerfSprint() *perfSprint {
 	}
 }
 
+// checker is a type of check that can be performed by the analyzer.
+// A checker can activate or deactivate specific rules.
+type checker = string
```

Comment:
Why a type alias ?

Reply:
You mean versus:
-  a string
- or versus `type checker string` (without the =)

I wanted a type, but I wanted something that didn't require type juggling when passing constants as flags

As I said in my first comment, I can remove everything and use string directly.

Reply:
Let's use a string then.

(Or ok for `type checker string` and I do not think you need type juggling, just defining the constants with the right type and using them...)

Reply:
I'm using a string now